### DISCUSSION TITLE
중고거래 페이지에서 책 판매 시세 확인 관련 문의

### DISCUSSION THREAD

**Discussion by JangGusWjd** [MajorIn-ABO/MajorIn_FE#20] [Ideas]
중고거래 페이지에서 오른쪽에 책 판매 시세에 대한 기능을 넣을려고 하는데
1. 책 제목, 저자, 출판사 입력
2. 확인 누르면 판매 금액 시세 확인

이런 과정인데 판매 금액의 평균을 보여주게 할까 아니면 최저금액, 최고금액을 나타나게 할까?
그리고 예를 들어 컴퓨터구조, 컴퓨터구조-개정판 이러면 검색된 리스트의 금액을 다 보여주는게 좋을까?

**Comment by cho-ja-young** [comment]
시세 관련해서는 최고금액, 최저금액만 보여주는 게 좋을 것 같고
리스트는 다 보여주는 게 좋을 것 같아!!
몇판인지 모르거나 그런거 신경 안쓰고 검색해보는 사람들도 있을 것 같아서!!

**Comment by wlsl10** [comment]
웅 나도 리스트는 다 보여주는 게 최신판 사기에도 좋을 것 같아!
그리고 시세는 최고 금액은 알 필요가 없을 것 같아서 구매자 입장에서 생각했을 때 최저 금액을 보여주는 게 좋을 것 같아! 
최저나 평균 둘다 좋을 것 같은데 시간 상 최저!
